data.json

```diff
@@ -532,5 +532,11 @@
     "githubUsername": "adityaoberai",
     "imageUrl": "https://user-images.githubusercontent.com/31401437/106563296-974b8f80-6551-11eb-8753-a13ea9e944c2.png",
     "issueNumber": 304
+  },
+      {
```

Comment:
I think this brace should be lined up with the rest :nerd_face:

Reply:
You are right ,  I fixed it and I have just opened a new issue and  PR to keep it organized  😄